components/reusable-components/pagination/HomePagination.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import { fetchItems, Item } from './lib/data';
+import StructurePagination from './StructurePagination';
+import Dropdown from './Dropdown';
+import styles from './HomePagination.module.scss';
+
+const HomePagination: React.FC = () => {
+  const [items, setItems] = useState<Item[]>([]);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(1);
+  const [itemsPerPage, setItemsPerPage] = useState(25);
+
+  const options = [25, 50, 100];
```

Comment:
the name should be all uppercase letter, it is a convention, also use more descriptive name